integration/test/controlflow/controlflow_test.go

```diff
@@ -36,38 +36,29 @@ func Test_Finalizer_Integration_Controlflow(t *testing.T) {
 
 	ctx := context.Background()
 
-	var r *testresource.Resource
+	var resource *testresource.Resource
 	{
 		c := testresource.Config{
 			Name: resourceName,
 		}
 
-		r, err = testresource.New(c)
+		resource, err = testresource.New(c)
 		if err != nil {
 			t.Fatalf("err == %v, want %v", err, nil)
 		}
 	}
 
-	var w wrapper.Interface
+	var harness wrapper.Interface
```

Comment:
That is not cool. I made specific PRs to rename this because it did not make sense. Please keep it as is in master.

Reply:
Fixed, PTAL.

Reply:
Please make sure you revert all changes in the integration tests except your necessary import changes. You change naming and leave methods behind that are not even used anymore. Please take your time and sort this out. When I can assist or when anything is unclear about the desired state let's please talk in person.